bindings/java/src/org/sleuthkit/datamodel/SleuthkitCase.java

```diff
@@ -2956,7 +2956,11 @@ public DerivedFile addDerivedFile(String fileName, String localPath,
 			connection.beginTransaction();
 
 			final long parentId = parentFile.getId();
-			final String parentPath = parentFile.getParentPath() + parentFile.getName() + '/'; //NON-NLS
+			String parentParentPath = parentFile.getParentPath();
+			if(!parentParentPath.endsWith("/")) {
```

Comment:
The slashes are OS dependent. I think that the tsk_files.parent_path should have / as path separators irrespective of the OS.